### PR TITLE
[code review only] feat(BBD-825): save cart status in store

### PR DESCRIPTION
**Pull request by treeforever**
currently does:
- save tracker info in cart state
- create cart and save cart id in state
- add to cart
- use persist to save to browser

doesn't do: 
- transform when writing to and reading from browser
- API calls (code in saga is stubbed)
- consolidate data in the cart

### CODE REVIEW

File: src/core/configuration.ts [hunk 1]
```diff
@@ -221,7 +221,7 @@ namespace Configuration {
     location?: Configuration.Recommendations.Location;
 
     /**
-     * Product ID field as used in recommendations.
+     * Product ID field as used in recommendations.`
```
Comment (zfcb) [inline src/core/configuration.ts]: Quasiquote added here?

File: src/core/observer.ts [hunk 2]
```diff
@@ -147,6 +147,13 @@ namespace Observer {
             orderHistory: emit(Events.ORDER_HISTORY_UPDATED)
           },
 
+          cart: {
+            content: {
+              cartId: emit(Events.CART_ID_UPDATED),
+              quantity: emit(Events.CART_UPDATED)
```
Comment (zfcb) [inline src/core/observer.ts]: Rename event to cart quantity updated

File: src/core/adapters/cart.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  export const transformToBrowser = (state: Store.Cart, reducerKey: string): any => state;
+
+  // tslint:disable-next-line:max-line-length
+  export const transformFromBrowser = (incomingState: Store.Cart): Store.Cart => {
+    return incomingState;
+  };
+
+  export const calculateQuantity = (items: any[]) =>
+    items.reduce((acc, item) => {
+      return acc + item['quantity'];
```
Comment (zfcb) [inline src/core/adapters/cart.ts]: I think calculateTotalQuantity would be a better name

File: src/core/adapters/configuration.ts [hunk 4]
```diff
@@ -33,7 +33,7 @@ namespace Adapter {
           page: {
             ...PageReducer.DEFAULTS,
             sizes: Adapter.extractPageSizes(config, PageReducer.DEFAULT_PAGE_SIZE)
-          },
+          }
```
Comment (zfcb) [inline src/core/adapters/configuration.ts]: Comma removed here?

Comment (treeforever) [reply src/core/adapters/configuration.ts]: why not?

**Pull request closed without merging**